includes/class-wc-siftscience-html.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * This class henarates html to display subsections stated in the admin file.
+ * This class henarates displayes html for subsections as stated in the admin file.
```

Comment:
This class generates html for subsections as...

Reply:
agreed